packages/@uppy/provider-views/src/ProviderView/ProviderView.jsx

```diff
@@ -347,11 +355,15 @@ export default class ProviderView extends View {
     const { files, folders, filterInput, loading, currentSelection } = this.plugin.getPluginState()
     const { isChecked, toggleCheckbox, recordShiftKeyPress, filterItems } = this
     const hasInput = filterInput !== ''
+    const pluginIcon = () => {
+      return this.plugin.icon?.() || defaultPickerIcon()
+    }
```

Comment:
I don't really know (p)react, do I need to wrap it like this for reactivity or can I just do `const pluginIcon = this.plugin.icon || defaultPickerIcon `

Reply:
I think you can do the latter, yes, since that's how we pass it in the Dashboard.